### PR TITLE
Release 1.16.0

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR is a mechanical PR to be able to merge to `master`.

When we have landed this branch, a release can be cut.

Details:

- Have updated the rake task to "fetch new certs" - since their URL had changed. No change to the provided certs.
- Have created a new CHANGELOG

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -15,57 +15,62 @@ PATH
...
       public_suffix (>= 2.0.2, < 5.0)
     ast (2.4.1)
-    async (1.26.2)
-      console (~> 1.0)
+    async (1.28.9)
+      console (~> 1.10)
       nio4r (~> 2.3)
       timers (~> 4.1)
-    async-http (0.52.5)
+    async-http (0.54.1)
```
Comment (olleolleolle) [inline Gemfile.lock:31]: Did something change between these, that I want to update to? @ioquatix 

```
Failure/Error:
  def initialize(endpoint, protocol: endpoint.protocol, scheme: endpoint.scheme, authority: endpoint.authority, retries: DEFAULT_RETRIES, connection_limit: DEFAULT_CONNECTION_LIMIT)
  	@endpoint = endpoint
  	@protocol = protocol
  	
  	@retries = retries
  	@pool = make_pool(connection_limit)
  	
  	@scheme = scheme
  	@authority = authority
  end

ArgumentError:
  wrong number of arguments (given 4, expected 1)
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/client.rb:48:in `initialize'
```

Comment (olleolleolle) [reply Gemfile.lock:31]: Sorry, here are long backtrace notes:

```
  wrong number of arguments (given 4, expected 1)
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/client.rb:48:in `initialize'
./vendor/bundle/ruby/2.5.0/gems/webmock-3.9.1/lib/webmock/http_lib_adapters/async_http_client_adapter.rb:43:in `new'
./vendor/bundle/ruby/2.5.0/gems/webmock-3.9.1/lib/webmock/http_lib_adapters/async_http_client_adapter.rb:43:in `initialize'
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/internet.rb:57:in `new'
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/internet.rb:57:in `client_for'
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/internet.rb:45:in `block in call'
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/internet.rb:44:in `fetch'
./vendor/bundle/ruby/2.5.0/gems/async-http-0.54.1/lib/async/http/internet.rb:44:in `call'
./vendor/bundle/ruby/2.5.0/gems/async-http-faraday-0.9.0/lib/async/http/faraday/adapter.rb:69:in `block (2 levels) in call'
./vendor/bundle/ruby/2.5.0/gems/async-http-faraday-0.9.0/lib/async/http/faraday/adapter.rb:97:in `with_timeout'
./vendor/bundle/ruby/2.5.0/gems/async-http-faraday-0.9.0/lib/async/http/faraday/adapter.rb:68:in `block in call'
./vendor/bundle/ruby/2.5.0/gems/async-1.28.9/lib/async/task.rb:265:in `block in make_fiber'
```

Comment (olleolleolle) [reply Gemfile.lock:31]: Aha, I was overlooking that webmock had been updated.

**Review by olleolleolle — COMMENTED** [review]
Well, now I feel good that there was a protective lock on the `master` branch.

**Comment by olleolleolle** [comment]
Labeling "invalid" in order not to keep it in the generated Markdown changelog.

**Comment by douglasmiller** [comment]
@olleolleolle, is there any expected date that this will be merged/released?

**Comment by olleolleolle** [comment]
@douglasmiller What's waiting is that the current rules are such that each PR to be merged to the default branch needs at least one member review:

bild

(That's the "branch protection" feature in GitHub.)

**Review by skywinder — APPROVED** [review]
hey, everything is gr8 sorry for delay

**Pull request merged**